src/get-started/quickstart.md

```diff
@@ -74,8 +88,9 @@ $ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
 ```
 
 {:.runner-flink}
-``` 
-TODO BEAM-899
+```
+$ mvn compile exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
```

Comment:
Unfortunately not, the Flink Runner does not automatically pick up class files and jars like the Dataflow runner does. You need to create a jar that contains all the dependencies (which a `mvn package` does because we have maven shade in the examples archetype) and then run it as you would run a Flink job, i.e. something like `bin/flink run -c my.WordCountClass my-jarfile.jar -my-parameters`.

Also, there is a bug right now which requires users to place the jar into the lib folder of the Flink installation. Something with Flink not using the proper Classloader in a certain place.

I can still put that in, though. What do you think?